plugins/pulp_ostree/plugins/lib.py

```diff
@@ -229,6 +223,158 @@ def pull_local(self, path, refs):
                 'refs': lib.GLib.Variant('as', tuple(refs))
             })
 
-        repository = lib.OSTree.Repo.new(fp)
-        repository.open(None)
-        repository.pull_with_options(url, options, None, None)
+        self.open()
+        self.impl.pull_with_options(url, options, None, None)
+
+
+class Remote(object):
+    """
+    Represents and OSTree remote repository.
```

Comment:
"Represents an OSTree..."

I'm not sure why GitHub didn't keep my last comment in-line. Hopefully this works out.